pkg/sensors/manager.go

```diff
@@ -349,9 +349,10 @@ type tracingPolicyAdd struct {
 }
 
 type tracingPolicyDelete struct {
-	ctx     context.Context
-	name    string
-	retChan chan error
+	ctx       context.Context
+	name      string
+	namespace string
```

Comment:
Please use a `collectionKey` type here. This will force us to update all the callers.

You would need for example to update https://github.com/cilium/tetragon/blob/25a0050cda943d8744b3f846d8c4a82219bce2da/pkg/sensors/manager.go#L204-L208

Reply:
Just a clarification, do you mean to add a `collectionKey` type or replacing the value of `name` altogether? Example:

```
type tracingPolicyAdd struct {
	ctx     context.Context
	name string
	ck      collectionKey
	tp      tracingpolicy.TracingPolicy
	retChan chan error
}
```

or 

```
type tracingPolicyAdd struct {
	ctx     context.Context
	ck      collectionKey
	tp      tracingpolicy.TracingPolicy
	retChan chan error
}
```